modules/netaddress.go

```diff
@@ -36,10 +36,10 @@ func (na NetAddress) IsLoopback() bool {
 	}
 
 	host := na.Host()
-	if ip := net.ParseIP(host); ip != nil && ip.IsLoopback() {
+	if ip := net.ParseIP(host); ip != nil && (ip.IsLoopback() || ip.IsUnspecified()) {
```

Comment:
This is confusing as a caller of `IsLoopback` would not expect it to return true for unspecified addresses. `IsLoopback` seems to be used as a check for validity. It seems to me it would be better to add
```go
ip := net.ParseIP(na.Host())
if ip.IsLoopback() || ip.IsUnspecified{
    return false
}
```
to the `NetAddress.IsValid` method.